x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/SearchApplicationTemplate.java

```diff
@@ -28,7 +27,8 @@
 
 /**
  * Search template included in a {@link SearchApplication}. It will be used for searching using the
- * {@link QuerySearchApplicationAction}, overriding the parameters included on it via {@link Request}
+ * {@link QuerySearchApplicationAction}, overriding the parameters included on it via
+ * {@link org.elasticsearch.xpack.application.search.action.SearchApplicationSearchRequest}
```

Comment:
nit: let's import the SearchApplicationSearchRequest to avoid requiring the package name?